pkgs/development/compilers/cudatoolkit/stdenv.nix

```diff
@@ -1,10 +1,17 @@
-{ overrideCC
+{ nixpkgsStdenv
+, nvccCompatibleStdenv
+, overrideCC
 , wrapCCWith
-, stdenv
-, compatibleStdenv
 }:
 
-overrideCC stdenv (wrapCCWith rec {
-  cc = compatibleStdenv.cc.cc;
-  libcxx = stdenv.cc.cc.lib;
+overrideCC nixpkgsStdenv (wrapCCWith {
+  cc = nvccCompatibleStdenv.cc.cc;
+
+  # This option is for libcxx, but we (ab)use it for gcc's libstdc++. Note that
```

Comment:
"This option is for clang" typo?

Reply:
No, I meant it this way, but maybe "for clang" is clearer?

Reply:
oh I didn't realize that libcxx was a clang-specific library. I assumed that gcc also called it libcxx. Perhaps "for clang's libcxx" would be clearest?

Reply:
(it also "rhymes" with the end of the sentence: "gcc's libstdc++")